custom_components/ui_lovelace_minimalist/lovelace/ulm_templates/card_templates/cards/card_media_player.yaml

```diff
@@ -94,20 +116,16 @@ card_media_player:
             card:
               type: "custom:button-card"
               entity: "[[[ return entity.entity_id ]]]"
-              variables: >
-                [[[
-                  let vars = variables;
-
-                  if(variables.ulm_card_media_player_enable_popup) {
-                    vars.ulm_custom_popup = {
-                      'template': 'popup_media_player_infos',
-                      'popup_variables': {
-                        'ulm_popup_media_player_entity': entity.entity_id,
-                      }
-                    };
-                  }
-                  return vars;
-                ]]]
+              variables:
+                ulm_icon_tap_action: "[[[ return variables.ulm_icon_tap_action; ]]]"
+                ulm_icon_tap_navigate_path: "[[[ return variables.ulm_icon_tap_navigate_path; ]]]"
+                ulm_icon_hold_action: "[[[ return variables.ulm_icon_hold_action; ]]]"
+                ulm_icon_hold_navigate_path: "[[[ return variables.ulm_icon_hold_navigate_path; ]]]"
+                ulm_icon_double_tap_action: "[[[ return variables.ulm_icon_double_tap_action; ]]]"
+                ulm_icon_double_tap_navigate_path: "[[[ return variables.ulm_icon_double_tap_navigate_path; ]]]"
+                ulm_input_select_option: "[[[ return variables.ulm_input_select_option; ]]]"
+                ulm_input_select: "[[[ return variables.ulm_input_select; ]]]"
+                ulm_card_media_player_enable_popup: "[[[ return variables.ulm_card_media_player_enable_popup; ]]]"
```

Comment:
branch not up to date

Reply:
See pr #878

Reply:
@matrixx567 can you point me in the right direction on how to resolve this?